src/extensions/default/AutoUpdate/main.js

```diff
@@ -39,7 +39,8 @@ define(function (require, exports, module) {
         StateHandlerModule      = require("StateHandler"),
         MessageIds              = require("MessageIds"),
         UpdateStatus            = require("UpdateStatus"),
-        UpdateInfoBar           = require("UpdateInfoBar");
+        UpdateInfoBar           = require("UpdateInfoBar"),
+        HealthLogger            = brackets.getModule("utils/HealthLogger");
```

Comment:
@niteskum for code clarity, please keep all the getModules and requires clubbed together, separately.